src/app/dataset-view/additional-components/metadata-component/components/stepper-navigation/stepper-navigation.component.ts

```diff
@@ -15,9 +15,9 @@ import { SetPollingSourceSection } from "src/app/shared/shared.types";
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class StepperNavigationComponent {
-    @Input() public nextStep: MaybeNull<SetPollingSourceSection>;
-    @Input() public prevStep: MaybeNull<SetPollingSourceSection>;
-    @Input() public validStep: boolean;
+    @Input() public nextStep?: MaybeNull<SetPollingSourceSection>;
```

Comment:
Isn't maybe null enough?

Reply:
If you are looking for default value, just assign one, such as `= null;`

Reply:
Fixed

Reply:
Remove `?`